### PR TITLE
[#157462870] mock timers in controller tests

### PR DESCRIPTION
**Pull request by gunzip**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Could not get info from npm on <a href='https://www.npmjs.com/package/@types/lolex'><code>&#64;types/lolex</code></a></a>
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🐞 [#157462870](https://www.pivotaltracker.com/story/show/157462870): Fix i test che dipendono da Date.now() falliscono su CircleCi


New dependencies added: <a href='https://www.npmjs.com/package/@types/lolex'><code>&#64;types/lolex</code></a> and <a href='https://www.npmjs.com/package/lolex'><code>lolex</code></a>.


<h2><a href="http://github.com/sinonjs/lolex">lolex</a></h2>
<p>Author: Christian Johansen</p>
<p>Description: Fake JavaScript timers</p>
<p>Homepage: <a href="http://github.com/sinonjs/lolex">http://github.com/sinonjs/lolex</a></p>

<table>
  <thead><tr><th></th><th width="100%"></th></tr></thead>
  <tr><td>Created</td><td>over 3 years ago</td></tr><tr><td>Last Updated</td><td>1 day ago</td></tr><tr><td>License</td><td>BSD-3-Clause</td></tr><tr><td>Maintainers</td><td>4</td></tr><tr><td>Releases</td><td>24</td></tr><tr><td>Direct Dependencies</td><td></td></tr>
</table>
This README is too long to show.

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: lib/controllers/__tests__/messages.test.ts [hunk 1]
```diff
@@ -56,6 +56,18 @@ import {
   MessagePayloadMiddleware
 } from "../messages";
 
+import * as lolex from "lolex";
+
+// mock time (ie. created_at)
+// tslint:disable-next-line:no-let
+let clock: any;
```
Comment (cloudify) [inline lib/controllers/__tests__/messages.test.ts]: not used yet?

Comment (gunzip) [reply lib/controllers/__tests__/messages.test.ts]: what exactly ?

Comment (cloudify) [reply lib/controllers/__tests__/messages.test.ts]: that variable

Comment (gunzip) [reply lib/controllers/__tests__/messages.test.ts]: yes that's because it must be a global one to be shared by afterEach and beforeEach methods

**Pull request merged**